discovery-package/src/main/resources/zosmf-static-definition.yaml.template

```diff
@@ -14,7 +14,8 @@ services:
         - gatewayUrl: api/v1
           serviceRelativeUrl:
       authentication:
-          scheme: zosmf
+          scheme: httpBasicPassTicket
+          applid: IZUDFLT
```

Comment:
does this belong here?

Reply:
No, unintentional change from explaining something. Reverted to the original state now